lib/mayaUsd/fileio/primWriterRegistry.cpp

```diff
@@ -74,32 +157,57 @@ void UsdMayaPrimWriterRegistry::RegisterRaw(
 }
 
 /* static */
-UsdMayaPrimWriterRegistry::WriterFactoryFn
-UsdMayaPrimWriterRegistry::Find(const std::string& mayaTypeName)
+UsdMayaPrimWriterRegistry::WriterFactoryFn UsdMayaPrimWriterRegistry::Find(
+    const std::string&          mayaTypeName,
+    const UsdMayaJobExportArgs& exportArgs,
+    const MObject&              exportObj)
 {
     TfRegistryManager::GetInstance().SubscribeTo<UsdMayaPrimWriterRegistry>();
 
-    // unfortunately, usdTypeName is diff from the tfTypeName which we use to
-    // register.  do the conversion here.
-    WriterFactoryFn ret = nullptr;
```

Comment:
IDK what that erased comment about usdTypeName vs tfTypeName was about... I guess it is now irrelevant?